app/controllers/api/v0/events_controller.rb

```diff
@@ -16,12 +19,18 @@ def create
     render nothing: true, status: 201
   end
 
-  class KafkaData < Hash
+  class KafkaData < HashWithIndifferentAccess
```

Comment:
HashWithIndifferentAccess lets users read values from this object using either string or symbol keys.  This also happens to fix the problem where calling `super` without parens was giving us weird behavior before, because `HashWithIndifferentAccess` has a nicer constructor.